lib/rubocop/cop/rspec/pending.rb

```diff
@@ -65,6 +67,10 @@ def skip_node?(node)
         def skip_symbol?(symbol_node)
           symbol_node == SKIP_SYMBOL || symbol_node == PENDING_SYMBOL
         end
+
+        def rspec?(node)
+          node.nil? || (node.const_type? && node.const_name == 'RSpec')
```

Comment:
Would it be possible to instead use our usual `{(const nil? :RSpec) nil?}` node matcher?

Reply:
Possible, if match is performed on the send node, and not on the receiver (as `receiver` may return nil instead of ast::node). Updated.